wiki/Submission/en.md

```diff
@@ -1,150 +1,31 @@
-
-Moderators can give a symbol to the beatmap's thread based on their evaluation of the beatmap. This symbol is displayed next to its subject in the subforum and next to the title of its first post. The possible symbols a beatmap can have are (from best to worst):
-
--   ![Heart icon](/wiki/shared/icon/heart.gif "Heart icon") Heart / ![Fire icon](/wiki/shared/icon/flame.gif "Fire icon") Flame
--   ![Thinking icon](/wiki/shared/icon/bubble.gif "Thinking icon") Thought bubble (Bubble for short)
--   ![Star icon](/wiki/shared/icon/star.gif "Star icon") Star
--   No symbol
--   ![Heart-broken icon](/wiki/shared/icon/broken-heart.gif "Heart-broken icon") Heart-broken
--   ![Nuke icon](/wiki/shared/icon/nuke.gif "Nuke icon") Nuke
-    -   *Graveyard* if inactive for a month.
-
-#### ![Heart icon](/wiki/shared/icon/heart.gif "Heart icon") Heart
-
-Once a second Beatmap Nominator nominates a beatmap with a ![Thinking icon](/wiki/shared/icon/bubble.gif "Thinking icon") Thought Bubble symbol, he/she qualifies it and it is automatically given this symbol when moved to the Ranked/Approved Beatmaps forum.
+`Upload Beatmap...` from the `File` dropdown in the editor (shortcut: `Ctrl` + `Shift` + `U`) will open the "Beatmap Submission System" (BSS) window. This initially displays lists resources for users to ensure their maps are suitable for ranking, ask mapping-related questions, and find feedback for their beatmaps.
 
-#### ![Fire icon](/wiki/shared/icon/flame.gif "Fire icon") Flame
+### Warning Screen
 
-A Flame is given to beatmaps that do not have the technical problems that ![Nuke icon](/wiki/shared/icon/nuke.gif "Nuke icon") Nuked beatmaps have, and it is considered as Approved beatmaps because they are long and fit within the Ranking Criteria. Older Approval criteria included total score or difficulty, but only length is taken into account now. Those maps are typically difficult and made as a challenge to top players.
+If the beatmap a user is uploading is not already on the osu! website, the BSS window will state how many new uploads a user can publish. If the beatmap is nominated, the window warns that the nomination will be reset by updating the beatmap. If the beatmap is in the graveyard, the window warns that the beatmap will be raised to the pending beatmaps category.
 
-#### ![Thinking icon](/wiki/shared/icon/bubble.gif "Thinking icon") Thought Bubble
+![Beatmap Submission System warning screen](bss_warning.jpg "Beatmap Submission System warning screen")
```

Comment:
"Beatmap Submission System screen" should be good enough for what it's showing
and move this above the `Upload beatmap...` paragraph
and also put the image into an `img/` folder here + change link
and also it's .png not jpg